operator_config.py

```diff
@@ -55,7 +55,7 @@
         "required": False,
         "type": "boolean",
         "coerce": bool,
-        "default": False,
+        "default": True,
```

Comment:
No! This absolutely 100% should be `False`. Read the docs: https://cloud.google.com/run/docs/configuring/cpu-allocation#pricing-impact 

This also seems unrelated to the `env` config variable.